eva-lib/src/main/java/uk/ac/ebi/eva/lib/config/EvaProperty.java

```diff
@@ -0,0 +1,130 @@
+package uk.ac.ebi.eva.lib.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+@ConfigurationProperties(prefix = "eva")
+@Component
+public class EvaProperty {
+
+//    eva.mongo.host=@eva.mongo.host@
```

Comment:
An issue here is that it's not as straightforward as autoconfigured MongoDbFactory, because a basic MongoClient is created [here](https://github.com/EBIvariation/eva-ws/blob/540c5b9cb96a85188604b94d016724632962ad31/eva-lib/src/main/java/uk/ac/ebi/eva/lib/utils/DBAdaptorConnector.java#L87).